.github/workflows/protobuf-reprolang.yml

```diff
@@ -0,0 +1,37 @@
+name: Generated code is up to date
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/**'
+      - 'docs/**'
+      - 'bindings/**'
+      - 'scip.proto'
+      - 'buf**'
+      - '.tool-versions'
+      - 'dev/proto-generate.sh'
+      - 'dev/proto-generate-in-docker.sh'
+      - 'Dockerfile.bindings'
+      - 'cmd/scip/tests/reprolang/**'
+
+jobs:
+  gen-up-to-date:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Log in to the Container registry
+        uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - run: docker pull ghcr.io/sourcegraph/scip:latest || echo "no suitable cache"
```

Comment:
Why is it `scip-bindings-env` everywhere else, but just `scip` here?

Reply:
Ah, last place where I forgot to update it.
Frankly I don't think this helps at all, but I will still update the image once I'm back at the computer